src/types/server-transactions.ts

```diff
@@ -0,0 +1,243 @@
+import { AssetType, ScamInfoType } from './account';
+import { NftEnumType } from './tokens';
+
+export interface ScResultType {
+  callType: string;
+  gasLimit: number;
+  gasPrice: number;
+  nonce: number;
+  prevTxHash: string;
+  hash: string;
+  originalTxHash: string;
+  receiver: string;
+  sender: string;
+  timestamp: number;
+  value: string;
+  data?: string;
+  returnMessage?: string;
+}
+
+export interface TransactionTokensType {
+  esdts: string[];
+  nfts: string[];
+}
+
+export enum TxActionsEnum {
+  // esdtNft category
+  transfer = 'transfer',
+  // stake category
+  delegate = 'delegate',
+  stake = 'stake',
+  unDelegate = 'unDelegate',
+  stakeClaimRewards = 'claimRewards',
+  reDelegateRewards = 'reDelegateRewards',
+  withdraw = 'withdraw',
+  // mex category
+  claimLockedAssets = 'claimLockedAssets',
+  swapTokensFixedInput = 'swapTokensFixedInput',
+  swapTokensFixedOutput = 'swapTokensFixedOutput',
+  swap = 'swap',
+  addLiquidity = 'addLiquidity',
+  addLiquidityProxy = 'addLiquidityProxy',
+  removeLiquidity = 'removeLiquidity',
+  removeLiquidityProxy = 'removeLiquidityProxy',
+  enterFarm = 'enterFarm',
+  enterFarmProxy = 'enterFarmProxy',
+  enterFarmAndLockRewards = 'enterFarmAndLockRewards',
+  enterFarmAndLockRewardsProxy = 'enterFarmAndLockRewardsProxy',
+  exitFarm = 'exitFarm',
+  exitFarmProxy = 'exitFarmProxy',
+  claimRewards = 'claimRewards',
+  claimRewardsProxy = 'claimRewardsProxy',
+  compoundRewards = 'compoundRewards',
+  compoundRewardsProxy = 'compoundRewardsProxy',
+  wrapEgld = 'wrapEgld',
+  unwrapEgld = 'unwrapEgld',
+  unlockAssets = 'unlockAssets',
+  mergeLockedAssetTokens = 'mergeLockedAssetTokens',
+  ping = 'ping'
+}
+
+export enum TxActionCategoryEnum {
+  esdtNft = 'esdtNft',
+  mex = 'mex',
+  stake = 'stake',
+  scCall = 'scCall'
+}
+
+export interface TokenArgumentType {
+  type: NftEnumType | 'FungibleESDT';
+  name: string;
+  ticker: string;
+  collection?: string;
+  identifier?: string;
+  token?: string;
+  decimals: number;
+  value: string;
+  providerName?: string;
+  providerAvatar?: string;
+  svgUrl?: string;
+}
+
+export interface TxActionType {
+  category: string;
+  name: TxActionsEnum;
+  description?: string;
+  arguments?: { [key: string]: any };
+}
+
+export interface UnwrapperType {
+  token?: TokenArgumentType[];
+  tokenNoValue?: TokenArgumentType[];
+  tokenNoLink?: TokenArgumentType[];
+  address?: string;
+  egldValue?: string;
+  value?: string;
+  providerName?: string;
+  providerAvatar?: string;
+}
+
+export enum TransactionOperationActionType {
```

Comment:
prepend all enums with Enum instead of Type. (rename with F2)
ex: `TransactionOperationActionType` -> `TxOperationActionEnum`